statsmodels/regression/tests/test_theil.py

```diff
@@ -27,7 +25,7 @@ def setup_class(cls):
                                 "theil_textile_predict.csv")
         cls.res_predict = pd.read_csv(filepath, sep=",")
 
-        names = "year	lconsump	lincome	lprice".split()
+        "year	lconsump	lincome	lprice".split()
```

Comment:
those would be better `#noqa`
assigning to "names" or "columns" is more informative about what the string is

Reply:
or commenting it out as purely information that is not code

Reply:
Should just be a comment.